### DISCUSSION TITLE
Read-only access internal repository?

### DISCUSSION THREAD

**Discussion by dds-anakamura** [community/community#64871] [Enterprise] [answered]
### Select Topic Area

Question

### Body

We have a GitHub Enterprise account with organization repository. I want to provide read-only access to individuals within the workplace without requiring that they have enterprise licenses. How would I go about setting this up?

**Comment by byrneh** [comment, marked as answer]
You can't,  they must be a member of the enterprise and will then get read only access to internal repositories.
They will consume an enterprise license

**Reply by dds-anakamura** [reply]
Thanks for the quick response. This is really unfortunate and a barrier to some of the great uses of GitHub.

**Comment by GabyCT** [comment]
In a private repository, repository owners can only grant write access to collaborators. Collaborators can’t have read-only access to repositories owned by a user account. More information about permission levels for a repository owned by a user account can be found here:

https://help.github.com/articles/permission-levels-for-a-user-account-repository/
